itou/utils/settings_context_processors.py

```diff
@@ -18,6 +18,8 @@ def expose_settings(request):
             # On all stats pages the help button should redirect to the C2 help page instead of the C1 help page.
             assistance_url = global_constants.PILOTAGE_ASSISTANCE_URL
 
+    base_template = "layout/_partial.html" if request.META.get("HTTP_HX_REQUEST") else "layout/base.html"
```

Comment:
grâce à `django-htmx`, tu peux utiliser `request.htmx is True`

Reply:
Oui, merci ! J'avais écrit ce code avant d'intégrer django-htmx, c'est un oubli.

Reply:
ne peut-on pas nommer cette partial `_htmx.html` ? Ca serait plus clair que de le mettre en commentaire dans la partial elle meme, a mon avis.

Reply:
Tu as tout à fait raison, ce sera plus clair.
Je suis allée plus loin en créant un dossier `layout/htmx` pour regrouper tous les gabarits de base utilisés par HTMX. J'ai donc renommé `layout/_partial.html` en `layout/htmx/base.html` et `layout/_base_modal.html` en `layout/htmx/modal.html`.